.circleci/config.yml

```diff
@@ -1,59 +0,0 @@
-version: 2.1
-
-jobs:
-  build:
-    working_directory: ~/slovensko-digital/navody.slovensko.digital
-
-    docker:
-      - image: circleci/ruby:2.7.2-node-browsers
```

Comment:
Tipujem ze testy failuju preto lebo tento image co pouzivalo circleci mal v sebe aj webdriver na selenium, zatial neviem ako to urobit cez GH actions.